### PR TITLE
Support setting cluster controller memory through feature flag

### PR DESCRIPTION
**Pull request by hmusum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: config-model/src/main/java/com/yahoo/vespa/model/application/validation/QuotaValidator.java [hunk 1]
```diff
@@ -33,9 +33,11 @@ public class QuotaValidator implements Validator {
...
     public void validate(Context context) {
         var zone = context.deployState().zone();
-        var exclusivity = new Exclusivity(zone, context.deployState().featureFlags().sharedHosts());
-        var tuning = new CapacityPolicies.Tuning(context.deployState().featureFlags().adminClusterArchitecture(),
-                                                 context.deployState().featureFlags().logserverNodeMemory());
+        var featureFlags = context.deployState().featureFlags();
+        var exclusivity = new Exclusivity(zone, featureFlags.sharedHosts());
+        var tuning = new CapacityPolicies.Tuning(featureFlags.adminClusterArchitecture(),
+                                                 featureFlags.logserverNodeMemory(),
+                                                 featureFlags.logserverNodeMemory());
```
Comment (arnej27959) [inline config-model/src/main/java/com/yahoo/vespa/model/application/validation/QuotaValidator.java:40]: did you really mean to use logserverNodeMemory twice here?

**Review by arnej27959 — DISMISSED** [review]
one possible issue here, rest looks good to me

File: config-model/src/main/java/com/yahoo/vespa/model/application/validation/QuotaValidator.java (hunk 1 above)
Comment (hmusum) [reply config-model/src/main/java/com/yahoo/vespa/model/application/validation/QuotaValidator.java]: No, this is wrong, thanks! Fixed

**Comment by hmusum** [comment]
@arnej27959 please take a look again

**Review by arnej27959 — APPROVED** [review]
Looks good to me!

**Pull request merged**